README.md

```diff
@@ -69,24 +69,25 @@ and may be chosen from a list of supported losses and regularizers, which includ
 
 Losses:
 
-* quadratic loss `quadratic`
-* hinge loss `hinge`
-* weighted hinge loss `weighted_hinge`
-* l1 loss `l1`
-* ordinal hinge loss `ordinal_hinge`
-* periodic loss `periodic`
+* quadratic loss `QuadLoss`
+* hinge loss `HingeLoss`
+* logistic loss `LogLoss`
+* weighted hinge loss `WeightedHinge`
+* l1 loss `L1Loss`
+* ordinal hinge loss `OrdinalHinge`
+* PeriodicLoss loss `PeriodicLoss`
 
 Regularizers:
 
-* quadratic regularization `quadreg`
-* constrained squared euclidean norm `constrained_quadreg`
-* l1 regularization `onereg`
-* no regularization `zeroreg`
-* nonnegative constraint `nonnegative` (eg, for nonnegative matrix factorization)
-* 1-sparse constraint `onesparse` (eg, for orthogonal NNMF)
-* unit 1-sparse constraint `unitonesparse` (eg, for k-means)
-* simplex constraint `simplex`
-* l1 regularization, combined with nonnegative constraint `nonneg_onereg`
+* QuadLoss regularization `QuadReg`
```

Comment:
let's stick with `quadratic` regularization, not `QuadLoss`. somehow the latter sounds like we're trying to copyright least squares.